client/lib/plans/constants.js

```diff
@@ -579,104 +578,104 @@ export const PLANS_LIST = {
+		getTitle: () => i18n.translate( 'All Premium features' ),
 	},
 
 	[ FEATURE_ADVANCED_CUSTOMIZATION ]: {
 		getSlug: () => FEATURE_ADVANCED_CUSTOMIZATION,
-		getTitle: () => 'Advanced customization', //PLANS A/B TEST: Translate if test passes
+		getTitle: () => i18n.translate( 'Advanced customization' ),
 	},
 
 	[ FEATURE_FREE_DOMAIN ]: {
 		getSlug: () => FEATURE_FREE_DOMAIN,
-		getTitle: () => 'Free custom domain', //PLANS A/B TEST: Translate if test passes
+		getTitle: () => i18n.translate( 'Free custom domain' ),
 	},
 
 	[ FEATURE_PREMIUM_THEMES ]: {
 		getSlug: () => FEATURE_PREMIUM_THEMES,
-		getTitle: () => 'Unlimited premium themes', //PLANS A/B TEST: Translate if test passes
+		getTitle: () => i18n.translate( 'Unlimited premium themes' ),
 	},
 
 	[ FEATURE_MONETISE ]: {
 		getSlug: () => FEATURE_MONETISE,
-		getTitle: () => 'Monetize your site with ads', //PLANS A/B TEST: Translate if test passes
+		getTitle: () => i18n.translate( 'Monetize your site with ads' ),
 	},
 
 	[ FEATURE_UPLOAD_THEMES_PLUGINS ]: {
 		getSlug: () => FEATURE_UPLOAD_THEMES_PLUGINS,
-		getTitle: () => 'Upload themes and plugins', //PLANS A/B TEST: Translate if test passes
+		getTitle: () => i18n.translate( 'Upload themes and plugins' ),
 	},
 
 	[ FEATURE_GOOGLE_ANALYTICS_SIGNUP ]: {
 		getSlug: () => FEATURE_GOOGLE_ANALYTICS_SIGNUP,
-		getTitle: () => 'Google Analytics', //PLANS A/B TEST: Translate if test passes
+		getTitle: () => i18n.translate( 'Google Analytics' ),
 	},
 
 	[ FEATURE_EMAIL_LIVE_CHAT_SUPPORT_SIGNUP ]: {
 		getSlug: () => FEATURE_EMAIL_LIVE_CHAT_SUPPORT_SIGNUP,
-		getTitle: () => 'Email and live chat support', //PLANS A/B TEST: Translate if test passes
+		getTitle: () => i18n.translate( 'Email and live chat support' ),
 	},
 
 	[ FEATURE_FREE_THEMES_SIGNUP ]: {
 		getSlug: () => FEATURE_FREE_THEMES_SIGNUP,
-		getTitle: () => 'Hundreds of free themes', //PLANS A/B TEST: Translate if test passes
+		getTitle: () => i18n.translate( 'Hundreds of free themes' ),
 	},
 
 	[ FEATURE_WP_SUBDOMAIN_SIGNUP ]: {
 		getSlug: () => FEATURE_WP_SUBDOMAIN_SIGNUP,
-		getTitle: () => 'WordPress.com subdomain', //PLANS A/B TEST: Translate if test passes
+		getTitle: () => i18n.translate( 'WordPress.com subdomain' ),
 	},
 
 	[ FEATURE_UNLIMITED_STORAGE_SIGNUP ]: {
 		getSlug: () => FEATURE_UNLIMITED_STORAGE_SIGNUP,
-		getTitle: () => 'Unlimited storage', //PLANS A/B TEST: Translate if test passes
+		getTitle: () => i18n.translate( 'Unlimited storage' ),
 	},
 
 	[ FEATURE_ADVANCED_SEO_TOOLS ]: {
 		getSlug: () => FEATURE_ADVANCED_SEO_TOOLS,
-		getTitle: () => 'Advanced SEO tools', //PLANS A/B TEST: Translate if test passes
+		getTitle: () => i18n.translate( 'Advanced SEO tools' ),
 	},
 
 	[ FEATURE_BACKUP_STORAGE_SPACE_UNLIMITED_SIGNUP ]: {
 		getSlug: () => FEATURE_BACKUP_STORAGE_SPACE_UNLIMITED_SIGNUP,
-		getTitle: () => 'Unlimited Backup Space', //PLANS A/B TEST: Translate if test passes
+		getTitle: () => i18n.translate( 'Unlimited Backup Space' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Unlimited Backup Storage Space' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/remove/plans-signup-test)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).